components/net/http_loader.rs

```diff
@@ -1735,6 +1718,36 @@ impl Drop for ResponseEndTimer {
     }
 }
 
+fn prompt_user_for_credentials(
+    embedder_proxy: &Mutex<EmbedderProxy>,
+) -> Option<PromptCredentialsInput> {
+    let proxy = embedder_proxy.lock().unwrap();
+
+    let Ok((ipc_sender, ipc_receiver)) = ipc::channel() else {
+        error!("couldn't create ipc sender and receiver");
```

Comment:
should I log the actual `Error` value?

Reply:
We can unwrap channel() errors. That's almost always going to be a resource starvation issue and the browser is going to crash anyways.